src/components/Disputes/PrecedenceCampaign/assets/PCModal.svg

```diff
@@ -0,0 +1 @@
041c0 0-4.925 4.867-9.631 7.263 0 0 14.258 4.49 18.012 9.802-7.601-2.39-20.584-1.495-23.71 4.961 8.679-.936 13.993 1.306 15.111 4.631 0 0-2.064 2.974-8.382 4.295-6.729 1.408-8.448 4.205-8.448 8.112s5.11 7.479 14.39 7.19c9.268-.297 34.545-4.12 36.456-51.943-1.911-4.497-25-13.679-41.526-13.578l.06 10.226z" fill="#FFCDBA"/><path fill-rule="evenodd" clip-rule="evenodd" d="M403.478 362.445h23.854l-1.502 7.859 9.421 7.504.689.547 6.504 150.539h-17.031l-11.622-146.601h-15.344l-37.143 146.601h-17.03l31.947-166.449h27.257z" fill="#7AC7DF"/><path fill-rule="evenodd" clip-rule="evenodd" d="M357.698 296.662a23.742 23.742 0 0113.58-4.274l19.783-.005-.009.008h8.2v8.456l12.95-8.456h13.637l-.009-.008 10.342.003a23.762 23.762 0 0118.611 9.012l11.505 14.552-12.901 9.708-9.577-11.564-8.559 48.35h-59.03l-9.315-52.609-16.7 12.07-10.062-12.991 17.554-12.252z" fill="#242424"/><path fill-rule="evenodd" clip-rule="evenodd" d="M435.251 362.444l24.979-28.523-6.843-8.263 12.901-9.708 9.615 12.161a9.945 9.945 0 01-.872 13.311l-28.206 27.387-10.886 9.546-.688-.547-9.421-7.505 1.502-7.859h7.919zM289.043 279.848l4.556.805 6.707 1.185 2.278.404v5.891l31.115 25.286 6.445-4.505 10.062 12.99-10.76 7.771a9.95 9.95 0 01-12.668-.84l-34.323-32.512-3.635-2.999-4.323-3.569-1.967-1.622 6.513-8.285zM404.486 259.313s.969 3.982 4.1 6.232a4.444 4.444 0 012.404-.709c.426 0 .833.078 1.221.19 1.89.536 3.276 2.25 3.276 4.306 0 2.055-1.386 3.769-3.276 4.305v18.746l-.01.008-12.949 8.456v-20.21l-5.322 1.395c-2.82.651-5.515-1.49-5.515-4.384V266.98s9.712.014 16.071-7.667z" fill="#F5C5B6"/><path fill-rule="evenodd" clip-rule="evenodd" d="M328.503 536.274l15.771-7.38h17.03v12.197h-31.725a2.53 2.53 0 01-1.076-4.817zM409.643 536.274l15.77-7.38h17.031v12.197h-31.725a2.53 2.53 0 01-2.53-2.527 2.54 2.54 0 011.454-2.29zM403.711 250.856c3.082 0 6.184.911 8.791 2.608.427-.586.892-1.128 1.28-1.567 2.074-2.351 5.147-4.011 8.268-3.723 4.439.411 7.822 4.47 12.232 5.109 4.139.602 8.026-1.925 12.116-2.797 4.343-.925 9.218.209 12.281 3.421 3.063 3.211 3.887 8.546 1.435 12.247-3.17 4.796-10.313 5.309-15.586 3-5.264-2.308-9.286-6.677-13.706-10.358-2.869-2.384-6.146-4.587-9.868-4.841-2.326-.159-4.216.558-6.077 1.427 2.52 2.51 4.178 5.999 4.178 10.358 0 9.456-6.844 14.108-6.844 14.108v-6.211c1.89-.536 3.277-2.25 3.277-4.305 0-2.056-1.387-3.77-3.277-4.306a4.378 4.378 0 00-1.221-.19c-.892 0-1.706.265-2.404.709-3.131-2.251-4.1-6.233-4.1-6.233-6.358 7.682-16.071 7.668-16.071 7.668-3.005-12.869 7.696-16.124 15.296-16.124z" fill="#352C47"/><rect x=".11" y="366.713" width="178.142" height="174.378" rx="10" fill="#EFF2FF"/><defs><linearGradient id="paint0_linear" x1="-17.639" y1="292.18" x2="209.423" y2="79.977" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint1_linear" x1="229.406" y1="116.955" x2="297.801" y2="35.317" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient></defs></svg>
```

Comment:
should we optimize this svg?

Reply:
It's already optimized :D Github shows the whole svg wrapped but it's all in one line